.travis.yml

```diff
@@ -32,7 +32,7 @@ script:
 
 after_success:
   - pip install coveralls
-  - coveralls
+  - coveralls --rcfile=setup.cfg
```

Comment:
Can you please explain why is this required? It seemed to me like it was working before quite well.

Reply:
I prefer to be explicit if I am not using default values and default value for `coverage` rc file is `.coveragerc` (however in reality it just search for this files by checking several ones).
I can remove it if you want